src/api_client/youtube_client.py

```diff
@@ -170,13 +170,11 @@ def upload(self, file, lang):
 
         # todo refactor this to make lang more flexible
         if lang:
-            if len(self.ticket.languages) > 1:
-                if lang in self.translation_strings.keys():
-                    metadata['snippet']['title'] += ' - ' + self.translation_strings[lang]
-                else:
-                    raise YouTubeException('language not defined in translation strings')
+            if lang in self.translation_strings.keys():
```

Comment:
The Condition here is not required, because the number of Languages of this Ticket is already checked at calling-time and the supplied language-parameter is only set, if a translation-suffix is to be appended.